### PR TITLE
Added Simplified grid component.

### PR DESCRIPTION
**Pull request by manojkumar8400**
Added Simplified grid component in the component library
Screenshot from 2022-04-02 23-35-50
.

### CODE REVIEW

**Review by Chakravarthi0 — APPROVED** [review]
clean code, good to merge

File: documentation/simplified-grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+.container {
+    margin-top: 6.5rem;
+    padding: 1rem;
+}
+
+.grid-rows {
+    display: grid;
+    justify-items: center;
+    text-align: center;
```
Comment (himadri2110) [inline documentation/simplified-grid/grid.css:11]: You can use the shorthand `place-items: center` to vertically and horizontally center items

**Review by himadri2110 — APPROVED** [review]
Good to go.

File: documentation/simplified-grid/grid.css (hunk 1 above)
Comment (manojkumar8400) [reply documentation/simplified-grid/grid.css:11]: Thanx, I Will apply that.

**Pull request merged**